requirements.txt

```diff
@@ -0,0 +1 @@
+-r requirements/heroku.txt
```

Comment:
As per [my email to wagtail-dev last July](https://gist.github.com/gasman/0e6b5551104830e51e5f) (which I really should move forward with at some point...) I'm broadly in favour of abolishing the requirements/dev.txt / requirements/production.txt separation. However, wagtaildemo is a slightly special case, in that the priority is to make it work on lots of different setups, rather than to ensure that dev and production environments are consistent.

Practically, I think the only issue would be if the Heroku or production requirements involved a package that was awkward to install (e.g. requiring external C libraries) - in which case we shouldn't inflict that on all users who just want a simple local setup. If not, then this is just a theoretical concern, and I'd be happy to go ahead with merging the requirements files.